portlets/src/main/webapp/vue-app/programsOverview/components/ProgramsOverview.vue

```diff
@@ -55,6 +99,18 @@ export default {
     clickOnEmptyActionLink() {
       window.location.href = `${eXo.env.portal.context}/${eXo.env.portal.portalName}/contributions/challenges`;
     },
+    retrievePrograms() {
```

Comment:
Can you add preload.resource.rest in the porlet config please!

Reply:
done!